tooling/appveyor/appveyor.yml

```diff
@@ -0,0 +1,32 @@
+version: 1.2.11.1.{build}
```

Comment:
Other projects seem to be converging on 'ci' rather than 'tooling', e.g. yesterday's commit https://github.com/glennrp/libpng/commit/3676fd324a8920930f1ee66248e3119d30c7c9a2

Reply:
I've moved the directory from tooling/appveyor/ to ci/.

I originally thought a tooling/ directory would be good because it makes room for other tools such as static analyzers without muddying the root directory. However, if/when we get to that bridge we can cross it. If ci/ is more comfortable, I'm happy to go with that.